finalPrep/modZeroHero.js

```diff
@@ -56,29 +83,85 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 //   - luckyNumbers (array)
 //   - address (object with following key/values: number , street , state, zip)
 
+var scaryMonster = {
+  name: "CloverField",
+  smell: "salty garbage",
+  weight: 400000,
+  citiesDestroyed:["Los Angeles"],
+  luckyNumbers:[77, 856, 1020],
+  address: {
+      number: 743,
+      street: "Evergreen Terrace",
+      state: "Illinois",
+      zip: 93105
+ }
+}
+console.log(scaryMonster);
 
 // Create a new class called SuperHero
 // - Your class should have the following DYNAMIC values
-//   - name 
+//   - name
 //   - superpower
-//   - age 
+//   - age
 // - Your class should have the following STATIC values
 //   - archNemesis, assigned to "The Syntax Error"
 //   - powerLevel = 100
-//   - energyLevel = 50 
+//   - energyLevel = 50
+class SuperHero {
+  constructor(name, superpower, age) {
+    this.name = name;
+    this.superpower = superpower;
+    this.age = age;
+    this. archNemesis = "The Syntax Error";
```

Comment:
What a funny line to have an extra space! Hah- the space after the dot shouldn't keep this from working, but you'll develop an eye for catching extra spaces, missed indentation, and the like.